lib/type/integer.ex

```diff
@@ -7,3 +7,13 @@ end
 defmodule Antikythera.MilliSecondsSinceEpoch do
   use Croma.SubtypeOfInt, min: 0
 end
+
+defmodule Antikythera.MilliSecondsInGregorian do
+  @time_epoch_offset_milliseconds (:calendar.datetime_to_gregorian_seconds({{1970, 1, 1}, {0, 0, 0}}) * 1000)
+
```

Comment:
This blank line is unnecessary.

In this repository, constants and their getters are often written without blank lines.
Please refer to other modules.

Reply:
5e708bff7a4ab99e50a09b24af51cfb4eab157a3